### PR TITLE
Mixed mass matrix

### PR DESCRIPTION
**Pull request by alessiofumagalli**
the branch implement the mass matrix for a mixed method, the implementation is very similar to the mass matrix for primal methods but here we take into account the saddle-point structure of the matrix.

NOTE: carefully check the modification in grid.py, due to a warning suppression. On my laptop all tests were working but this modification might have decreases some performances. I think not.

### CODE REVIEW

**Comment by alessiofumagalli** [comment]
I need to revert the part related to the note, it causes a memory error. So I indeed created a performance break

File: src/porepy/numerics/vem/mass_matrix.py [hunk 1]
```diff
@@ -0,0 +1,327 @@
+"""
+Mass matrix classes for a discretization of a L2-mass bilinear form with constant test
+and trial functions for mixed methods (e.g. RT0, MVEM).
+
+The discretization takes into account cell volumes, porosity, time step and aperture,
+so that the mass matrix (shape (g.num_faces + g.num_cells)^2) has the following diagonal
+for the cell dof:
+g.cell_volumes * porosity * aperture / deltaT
```
Comment (keileg) [inline src/porepy/numerics/vem/mass_matrix.py]: deltaT is out, please update comment (and in the file where you copied the documentation from.

File: src/porepy/numerics/vem/mass_matrix.py [hunk 2]
```diff
@@ -0,0 +1,327 @@
...
+
+        Parameters:
+            g : grid, or a subclass, with geometry fields computed.
+            data: dictionary to store the data.
+
+        Returns:
+            matrix (sparse dia, self.ndof x self.ndof): Mass matrix obtained from the
+                discretization.
+            rhs (array, self.ndof):
+                Null right-hand side.
```
Comment (keileg) [inline src/porepy/numerics/vem/mass_matrix.py]: Null? Or zero array?

File: test/unit/test_mixed_mass_matrix.py [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+        g.compute_geometry()
+
+        specified_parameters = {"aperture": 1e-2, "mass_weight": 0.5}
+        data = pp.initialize_default_data(g, {}, "flow", specified_parameters)
+
+        discr = pp.MixedMassMatrix()
+        lhs, rhs = discr.assemble_matrix_rhs(g, data)
+
+        lhs_known = 1./1200 * \
+            np.array([[0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0.],
```
Comment (keileg) [inline test/unit/test_mixed_mass_matrix.py]: Black. Although I hate what it will do to this file.

**Comment by keileg** [comment]
I see no changes in grid.py

File: src/porepy/numerics/vem/mass_matrix.py (hunk 1 above)
Comment (alessiofumagalli) [reply src/porepy/numerics/vem/mass_matrix.py]: thanks, done

File: src/porepy/numerics/vem/mass_matrix.py (hunk 2 above)
Comment (alessiofumagalli) [reply src/porepy/numerics/vem/mass_matrix.py]: I put zero, null is misleading

**Pull request merged**